Source/WebCore/page/LocalFrame.cpp

```diff
@@ -1406,19 +1407,41 @@ void LocalFrame::setScrollingMode(ScrollbarMode scrollingMode)
 
 #if ENABLE(CONTENT_EXTENSIONS)
 
-static String generateResourceMonitorErrorHTML()
+static String generateResourceMonitorErrorHTML(bool mayUseLightApparance, bool mayUseDarkApparance)
```

Comment:
I think we should have this take an `OptionSet<ColorScheme>`. Which isn't possible at this moment since that enum is behind `#if ENABLE(DARK_MODE_CSS)`. But I think you can just remove that `#if`.

Reply:
Yeah, `#if ENABLE(DARK_MODE_CSS)` this is the reason I was away from using `ColorScheme`.

Reply:
You can go ahead and remove the `#if ENABLE(DARK_MODE_CSS)` around it.

Reply:
(appearance is also spelled incorrectly in quite a few places (Apparance), but that should go away once you start using `ColorScheme`)